core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -3517,6 +3517,7 @@ XDDkwd__(SUBQUERY_UNNESTING,			"ON"),
  XDDui1__(USTAT_MIN_ROWCOUNT_FOR_LOW_SAMPLE,    "1000000"),
  XDDui1__(USTAT_MIN_ROWCOUNT_FOR_SAMPLE,        "10000"),
   DDflt0_(USTAT_MODIFY_DEFAULT_UEC,             "0.05"),
+  DDflt0_(USTAT_NAHEAP_ESTIMATED_MAX,           "1.3"),  // estimated max memory allocation feasible with NAHEAP.
```

Comment:
Will add a comment. The unit is in GB.

On Thu, Jul 16, 2015 at 1:51 PM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/sqlcomp/nadefaults.cpp
> <https://github.com/apache/incubator-trafodion/pull/30#discussion_r34823772>
> :
>
> > @@ -3517,6 +3517,7 @@ XDDkwd__(SUBQUERY_UNNESTING,			"ON"),
> >   XDDui1__(USTAT_MIN_ROWCOUNT_FOR_LOW_SAMPLE,    "1000000"),
> >   XDDui1__(USTAT_MIN_ROWCOUNT_FOR_SAMPLE,        "10000"),
> >    DDflt0_(USTAT_MODIFY_DEFAULT_UEC,             "0.05"),
> > +  DDflt0_(USTAT_NAHEAP_ESTIMATED_MAX,           "1.3"),  // estimated max memory allocation feasible with NAHEAP.
>
> What units are used for this default?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/30/files#r34823772>.
>



-- 
Regards, --Qifan